uiplib/gui/settingsTab.py

```diff
@@ -118,13 +132,25 @@ def file_open_helper(appObj):
     appObj.root.update_idletasks()
 
 
+def file_open_helper1(appObj):
```

Comment:
Dont create a new function, modify the old one to take a parameter
you can give it appObj.fav_pics_folder
and edit the old helper call to giver it appObj.pics_folder

Cool?